latest/index.html

```diff
@@ -1,10 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Why is it removed?

Reply:
Since `latest` is symlink it cannot store additional files inside self.